### PR TITLE
fix: lint arm action in yaml

### PR DESCRIPTION
**Pull request by xzf0587**
https://dev.azure.com/msazure/Microsoft%20Teams%20Extensibility/_workitems/edit/18512573

### CODE REVIEW

File: templates/scenarios/ts/sso-tab/teamsapp.yml.tpl [hunk 1]
```diff
@@ -26,15 +26,32 @@ provision:
...
-    with:
-      subscriptionId: ${{AZURE_SUBSCRIPTION_ID}} # The AZURE_SUBSCRIPTION_ID is a built-in environment variable. TeamsFx will ask you select one subscription if its value is empty. You're free to reference other environment varialbe here, but TeamsFx will not ask you to select subscription if it's empty in this case.
-      resourceGroupName: ${{AZURE_RESOURCE_GROUP_NAME}} # The AZURE_RESOURCE_GROUP_NAME is a built-in environment variable. TeamsFx will ask you to select or create one resource group if its value is empty. You're free to reference other environment varialbe here, but TeamsFx will not ask you to select or create resource grouop if it's empty in this case.
+  - uses: arm/deploy  # Deploy given ARM templates parallelly.
+    with:
+      # The AZURE_SUBSCRIPTION_ID is a built-in environment variable.
+      # TeamsFx will ask you select one subscription if its value is empty.
+      # You're free to reference other environment varialbe here,
+      # but TeamsFx will not ask you to select subscription
+      # if it's empty in this case.
```
Comment (jidddddd) [inline templates/scenarios/ts/sso-tab/teamsapp.yml.tpl:35]: AZURE_SUBSCRIPTION_ID is a built-in environment variable, if its value is empty, TeamsFx will prompt you to select a subscription. Referencing other environment variables with empty values will skip the subscription selection prompt.

File: templates/scenarios/ts/sso-tab/teamsapp.yml.tpl [hunk 2]
```diff
@@ -26,15 +26,32 @@ provision:
...
+      subscriptionId: ${{AZURE_SUBSCRIPTION_ID}}
+      # The AZURE_RESOURCE_GROUP_NAME is a built-in environment variable.
+      # If its value is empty,
+      # TeamsFx will ask you to select or create one resource group
+      # You're free to reference other environment varialbe here,
+      # but TeamsFx will not ask you to select or create resource grouop
+      # if it's empty in this case.
+      resourceGroupName: ${{AZURE_RESOURCE_GROUP_NAME}}
```
Comment (jidddddd) [inline templates/scenarios/ts/sso-tab/teamsapp.yml.tpl:43]: same as above

File: templates/scenarios/ts/sso-tab/teamsapp.yml.tpl [hunk 3]
```diff
@@ -26,15 +26,32 @@ provision:
...
       templates:
-       - path: ./infra/azure.bicep # Relative path to this file
-         parameters: ./infra/azure.parameters.json # Relative path to this file. Placeholders will be replaced with corresponding environment variable before ARM deployment.
-         deploymentName: Create-resources-for-tab # Required when deploy ARM template
-      bicepCliVersion: v0.9.1 # Teams Toolkit will download this bicep CLI version from github for you, will use bicep CLI in PATH if you remove this config.
+        - path: ./infra/azure.bicep  # Relative path to this file
+          # Relative path to this file.
```
Comment (jidddddd) [inline templates/scenarios/ts/sso-tab/teamsapp.yml.tpl:46]: Relative path to this yml file.

File: templates/scenarios/ts/sso-tab/teamsapp.yml.tpl [hunk 4]
```diff
@@ -26,15 +26,32 @@ provision:
...
+          # Placeholders will be replaced with corresponding environment
+          # variable before ARM deployment.
+          parameters: ./infra/azure.parameters.json
+          # Required when deploy ARM template
```
Comment (jidddddd) [inline templates/scenarios/ts/sso-tab/teamsapp.yml.tpl:50]: "deploy" should be "deploying"

File: templates/scenarios/csharp/command-and-response/teamsapp.yml.tpl [hunk 5]
```diff
@@ -20,15 +20,30 @@ provision:
...
+  - uses: arm/deploy  # Deploy given ARM templates parallelly.
     with:
-      subscriptionId: ${{AZURE_SUBSCRIPTION_ID}} # The AZURE_SUBSCRIPTION_ID is a built-in environment variable. TeamsFx will ask you select one subscription if its value is empty. You're free to reference other environment varialbe here, but TeamsFx will not ask you to select subscription if it's empty in this case.
-      resourceGroupName: ${{AZURE_RESOURCE_GROUP_NAME}} # The AZURE_RESOURCE_GROUP_NAME is a built-in environment variable. TeamsFx will ask you to select or create one resource group if its value is empty. You're free to reference other environment varialbe here, but TeamsFx will not ask you to select or create resource grouop if it's empty in this case.
+      # AZURE_SUBSCRIPTION_ID is a built-in environment variable,
+      # if its value is empty, TeamsFx will prompt you to select a subscription.
+      # Referencing other environment variables with empty values
+      # will skip the subscription selection prompt.
+      subscriptionId: ${{AZURE_SUBSCRIPTION_ID}}
+      # AZURE_SUBSCRIPTION_ID is a built-in environment variable,
```
Comment (swatDong) [inline templates/scenarios/csharp/command-and-response/teamsapp.yml.tpl:30]: `AZURE_RESOURCE_GROUP_NAME` here? Apply to all other files as well.

Comment (xzf0587) [reply templates/scenarios/csharp/command-and-response/teamsapp.yml.tpl]: Thanks. It is fixed.

**Review by swatDong — APPROVED** [review]

**Pull request merged**